ee/migrations/0007_migrate_definitions_tags.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.5 on 2022-01-28 19:21
+from django.db import migrations
+
+from posthog.models.tagged_item import EnterpriseTaggedItem
+
+
+def forwards(apps, schema_editor):
+    # Create new event definition tags
+    EnterpriseEventDefinition = apps.get_model("ee", "EnterpriseEventDefinition")
+    for instance in EnterpriseEventDefinition.objects.raw(
+        "SELECT * FROM ee_enterpriseeventdefinition WHERE tags!='{}' AND tags IS NOT NULL"
+    ):
+        for tag in instance.tags:
+            new_tag = EnterpriseTaggedItem(content_object=instance, tag=tag, team_id=instance.team_id)
+            new_tag.save()
+
+    # Create new property definition tags
+    EnterprisePropertyDefinition = apps.get_model("ee", "EnterprisePropertyDefinition")
+    for instance in EnterprisePropertyDefinition.objects.raw(
+        "SELECT * FROM ee_enterprisepropertydefinition WHERE tags!='{}' AND tags IS NOT NULL"
+    ):
+        for tag in instance.tags:
+            new_tag = EnterpriseTaggedItem(content_object=instance, tag=tag, team_id=instance.team_id)
+            new_tag.save()
+
+
+def reverse(apps, schema_editor):
+    EnterpriseEventDefinition = apps.get_model("ee", "EnterpriseEventDefinition")
```

Comment:
I'm not sure this reverse does exactly what we want. I think if we wanted to roll this back, we would want to delete all `global_tags` where the object tagged is the type `EnterpriseEventDefinition`